init-tools.cmd

```diff
@@ -84,6 +85,9 @@ if not [%INIT_TOOLS_ERRORLEVEL%]==[0] (
 
 :: Create semaphore file
 echo Done initializing tools.
+if NOT exist "%BUILD_TOOLS_SEMAPHORE_DIR%" (
```

Comment:
Move this to the top as there there is another location that writes the semaphore file as well see line 34.

Reply:
I would also make int a singe line.

Reply:
@weshaggard updated